src/components/Biography/EditBiography/View.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from 'react';
+
+function EditBiography({ toggleView, biography, handleBiographyChange }) {
+    const [editBiography, setBiography] = useState({
+        newbiography: biography,
+    });
+
+
+    const getNewValue = (e) => {
+        
+       setBiography(editBiography => ({ ...editBiography, [e.target.name]: e.target.value }))
+
+    }
+
+    useEffect(() => { handleBiographyChange(editBiography) }, [editBiography])
```

Comment:
this part here is what i was talking about in my previous comment, for the sake of readability I won't repeat the comment, but try the suggested solution and remove this part of code from all of the components that don't need it